packages/serverless/ci/dind-nodejs-aws-jq/Dockerfile

```diff
@@ -45,6 +46,11 @@ RUN apk --update --no-cache add \
     rm -rf /root/.cache
 
 # Install glibc compatibility for alpine plus AWS cli
+# ----------------------------------------------------------------------------
+# This section installs the latest version of AWS CLI (v2) and glibc.
+# AWS CLI v2 is fetched directly from Amazon’s servers using a static
+# download link, which always provides the latest release of the CLI tool.
+# https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html#getting-started-install-instructions
```

Comment:
I have added this comment for clarify that it always fetches the AWS CLI v2, which is the latest available version now, see https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html